lib/stoplight.rb

```diff
@@ -20,6 +20,8 @@ class << self
     def default_error_notifier=(value)
       warn "[DEPRECATED] `Stoplight.default_error_notifier=` is deprecated. " \
         "Please use `Stoplight.configure { |config| config.error_notifier= }` instead."
+      raise Error::ConfigurationError, "Stoplight must be configured only once" if @config_provider
```

Comment:
`NIT`
Should we move the error message ("Stoplight must be configured only once") to a constant? Since we already use it here and in `config/config_provider.rb`.